mp/src/game/server/momentum/mom_lobby_system.cpp

```diff
@@ -809,6 +802,25 @@ void CMomentumLobbySystem::ReceiveP2PPackets()
 
             CSteamID fromWho = pMessage->m_identityPeer.GetSteamID();
 
+            const auto findIndex = m_mapLobbyGhosts.Find(fromWho.ConvertToUint64());
+
+            if (IsUserBlocked(fromWho))
+            {
+                if (findIndex != m_mapLobbyGhosts.InvalidIndex())
+                {
+                    // Sender was just blocked, remove their ghost
+                    GetLobbyMemberEntity(fromWho)->Remove();
+                    m_mapLobbyGhosts.RemoveAt(findIndex);
+                }
+
+                continue;
```

Comment:
No bueno. Need to call `pMessage->Dispose()` before this.

Reply:
Do you mean `Release()`?

Reply:
Yep, apologies I thought it was Dispose.